releases/release-5.0.3.md

```diff
@@ -0,0 +1,137 @@
+    - Fix the issue that `SELECT DISTINCT` converted to `Batch Get` causes incorrect results [#25320](https://github.com/pingcap/tidb/issues/25320)
+    - Fix the issue that backing off queries from TiFlash to TiKV cannot be triggered [#24600](https://github.com/pingcap/tidb/pull/24600)
+    - Fix the `index-out-of-range` error that occurs when checking `only_full_group_by` [#23839](https://github.com/pingcap/tidb/issues/23839))
+    - Fix the issue that queries with `TABLESAMPLE` on an empty table returns unexpected rows [#25257](https://github.com/pingcap/tidb/issues/25257)
+    - Fix the issue that the result of index join in correlated subqueries is wrong [#25799](https://github.com/pingcap/tidb/issues/25799)
+
++ TiKV
+
+    - Fix the wrong `tikv_raftstore_hibernated_peer_state` metric [#10330](https://github.com/tikv/tikv/issues/10330)
+    - Fix the wrong arguments type of the `json_unquote()` function in the coprocessor [#10176](https://github.com/tikv/tikv/issues/10176)
+    - Fix the issue that Backup & Restore reports the error of "file already exists" when TDE is enabled during the restore [#1179](https://github.com/pingcap/br/issues/1179)
+    - Skip clearing callback during graceful shutdown to avoid breaking ACID in some cases [#10353](https://github.com/tikv/tikv/issues/10353) [#10307](https://github.com/tikv/tikv/issues/10307)
+    - Fix a bug that the read index is shared for replica reads on a Leader [#10347](https://github.com/tikv/tikv/issues/10347)
+    - Fix the wrong function that casts `DOUBLE` to `DOUBLE` [#25200](https://github.com/pingcap/tidb/issues/25200)
++ PD
+
+    - Fix the data race issue that occurs when loading TTL configurations after the scheduler is started [#3771](https://github.com/tikv/pd/issues/3771)
+    - Fix a bug that the `is_learner` field of the `TIKV_REGION_PEERS` table in TiDB is incorrect [#3372](https://github.com/tikv/pd/issues/3372) [#24293](https://github.com/pingcap/tidb/issues/24293)
+    - Fix the issue that when all TiKV nodes in a zone are offline or down, PD does not schedule replicas to other zones [#3705](https://github.com/tikv/pd/issues/3705)
+    - Fix the issue that PD might get panic after the scatter Region scheduler is added [#3762](https://github.com/tikv/pd/pull/3762)
+
++ TiFlash
+
+    - Fix the issue that TiFlash keeps restarting because of the split failure
+    - Fix the potential issue that TiFlash cannot delete the delta data
+    - Fix a bug that TiFlash adds wrong padding for non-binary characters in the `CAST` function
+    - Fix the issue of incorrect results when handling aggregation queries with complex `GROUP BY` columns
+    - Fix the TiFlash panic issue that occurs under heavy write pressure
+    - Fix the panic that occurs when the right jon key is not nullalbe and the left join key is nullable
+    - Fix the potential issue that the `read-index` requests take a long time
+    - Fix the panic issue that occurs when the read load is heavy
+
```

Comment:
- Fix panic issue that occurs when `Date_Format(string_column)` is called with NULL values in the string column.